aspnetcore/tutorials/getting-started-with-swashbuckle.md

```diff
@@ -110,6 +110,9 @@ If using directories with IIS or a reverse proxy, set the Swagger endpoint to a
 
 Swagger provides options for documenting the object model and customizing the UI to match your theme.
 
+In the Startup class, add the following namespaces:
+[!code-csharp[](../tutorials/web-api-help-pages-using-swagger/samples/2.0/TodoApi.Swashbuckle/Startup2.cs?name=snippet_PreReqNamespaces)]
```

Comment:
If we accept this, use absolute path, not relative path. That is
`~/tutorials/web-api-help-pages-using-swagger/samples/2.0/TodoApi.Swashbuckle/Startup2.cs`

Reply:
A region named `snippet_PreReqNamespaces` will need to be defined in the *Startup2.cs* file. This is why the snippet doesn't display.

Reply:
> A region named snippet_PreReqNamespaces will need to be defined in the Startup2.cs file. This is why the snippet doesn't display.

Done in PR #13702

Reply:
I added the region to the _Startup2.cs_ file in this PR.

Reply:
Just now

Reply:
Did you check the build?

Reply:
Just checked, seems to be showing up fine now.